consul_exporter.go

```diff
@@ -235,7 +248,7 @@ func main() {
 		showVersion   = flag.Bool("version", false, "Print version information.")
 		listenAddress = flag.String("web.listen-address", ":9107", "Address to listen on for web interface and telemetry.")
 		metricsPath   = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
-		consulServer  = flag.String("consul.server", "localhost:8500", "HTTP API address of a Consul server or agent.")
+		consulServer  = flag.String("consul.server", "localhost:8500", "HTTP API address of a Consul server or agent. (prefix with https:// to connect over HTTPS)")
```

Comment:
Makes sense. What if no scheme is provided? Should this be backward compatible and prepend `http://` if no scheme is provided?